### DISCUSSION TITLE
Cant get the UseLoadingIndicator Working.

### DISCUSSION THREAD

**Discussion by lotoras** [nuxt/nuxt#24858] [Questions] [answered]
i want to use the useLoadingIndicator Composable to Start and Finish it on My Fetch Requests Manually. But i get the error message
useLoadingIndicator is not defined everytime I include the useLoadingIndicator Composable the following way

const loadingIndicator = useLoadingIndicator();

i am running nuxt on version 3.8.2 and my package.json looks like this

{
    "private": true,
    "type": "module",
    "scripts": {
        "build": "nuxt build",
        "dev": "nuxt dev --host 127.0.0.1",
        "generate": "nuxt generate",
        "preview": "nuxt preview"
    },
    "devDependencies": {
        "@nuxt/devtools": "latest",
        "@nuxtjs/i18n": "^8.0.0-rc.4",
        "nuxt": "^3.8.2",
        "sass": "^1.67.0",
        "typescript": "^5.2.2"
    },
    "dependencies": {
        "@fortawesome/fontawesome-free": "^6.4.2",
        "@lessworkjs/optional": "^1.0.1",
        "@nuxtjs/toast": "^3.3.1",
        "@pinia/nuxt": "^0.4.11",
        "bootstrap": "^4.6.2",
        "fortawesome": "^0.0.1-security",
        "lodash": "^4.17.21",
        "moment": "^2.29.4",
        "nuxt-storage": "^1.2.2",
        "pinia": "^2.1.6",
        "primevue": "^3.35.0",
        "resize-textarea-vue3": "github:SecureIT-Swiss/resize-textarea-vue3#master",
        "tempusdominus-bootstrap-4": "^5.39.2",
        "v-pagination-3": "^0.1.7",
        "vite-svg-loader": "^4.0.0",
        "vue-autonumeric": "^1.2.6",
        "vue-i18n": "^9.5.0",
        "vue-multiselect": "^3.0.0-beta.2",
        "vue-plain-pagination": "^0.3.0",
        "vue-search-select": "^3.2.0",
        "vue-toastification": "^2.0.0-rc.5",
        "vue3-easy-data-table": "^1.5.47"
    },
    "optionalDependencies": {
        "@esbuild/darwin-arm64": "^0.19.3"
    },
    "overrides": {
        "vue": "latest"
    }
}

**Comment by khudia** [comment]
Same error. useLoadingIndicator is not defined

**Comment by TheAlexLichter** [comment, marked as answer]
This is a feature that will be released with version 3.9.0. It is available on the nightly channel right now.

PR: https://github.com/nuxt/nuxt/pull/24010

**Reply by TheAlexLichter** [reply]
The feature is now available!
